### ISSUE TITLE
Make dev fails with latest Kong rockspec (1.1.0) due to dependency on lyaml

### ISSUE THREAD

**Issue by onematchfox** [Kong/kong-vagrant#110]
Error as follows:
```
Installing https://luarocks.org/lyaml-6.2.3-1.src.rock

Error: Failed installing dependency: https://luarocks.org/lyaml-6.2.3-1.src.rock - Could not find library file for YAML
  No file libyaml.a in /usr/local/lib
  No file libyaml.a in /usr/local/lib/x86_64-linux-gnu
  No file libyaml.so in /usr/local/lib
  No file libyaml.so in /usr/local/lib/x86_64-linux-gnu
  No file matching libyaml.so.* in /usr/local/lib
  No file matching libyaml.so.* in /usr/local/lib/x86_64-linux-gnu
  No file libyaml.a in /usr/lib
  No file libyaml.a in /usr/lib/x86_64-linux-gnu
  No file libyaml.so in /usr/lib
  No file libyaml.so in /usr/lib/x86_64-linux-gnu
  No file matching libyaml.so.* in /usr/lib
  No file matching libyaml.so.* in /usr/lib/x86_64-linux-gnu
You may have to install YAML in your system and/or pass YAML_DIR or YAML_LIBDIR to the luarocks command.
Example: luarocks install lyaml YAML_DIR=/usr/local
```

**Comment by Tieske** [comment]
How did you install Kong itself? The binaries?

**Comment by Tieske** [comment]
A, I see the reference to the vagrant repo. 

This is a typical in between issue when the binaries are out of sync with the Kong repo. Will be fixed when 1.1 is released.
